Assets/_Scripts/UserInputManager.cs

```diff
@@ -8,26 +8,26 @@ public class UserInputManager : MonoBehaviour {
 	// variable set in Inspector
 	public int playerNum;
 
-
-
 	// variables for player controller and GUI
 	[HideInInspector]
 	public Vector2 leftInput;
 	[HideInInspector]
 	public Vector2 rightInput;
 	[HideInInspector]
-	public int magicID_in = 1;
+	public int button_id = -1;
 
 	// controller input events:
 	public delegate void OnInput();
-	public event OnInput OnPressRBumper;
-	public event OnInput OnPressLBumper;
+	public event OnInput OnPressSubSkill;
```

Comment:
How can UserInputManager understand What a skill is?
It should be purely handle button event.